pkg/v22/resource/appmigration/create.go

```diff
@@ -12,7 +12,6 @@ import (
 	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
 	"github.com/giantswarm/errors/tenant"
 	"github.com/giantswarm/microerror"
-	"github.com/giantswarm/operatorkit/controller/context/resourcecanceledcontext"
```

Comment:
Not needed since this is not a CRUD resource. See https://github.com/giantswarm/cluster-operator/pull/756#discussion_r340463938.